frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Dropdown, Input } from '~/bundles/common/components/components';
+import { InputType } from '~/bundles/common/enums/input-type.enum';
+import { useCallback } from '~/bundles/common/hooks/hooks';
+import { useAppForm } from '~/bundles/common/hooks/use-app-form/use-app-form.hook';
+
+import styles from './styles.module.scss';
+
+const InputsContainer: React.FC = () => {
+    const { control, errors } = useAppForm({
+        defaultValues: { name: '', balance: '' },
+    });
+
+    const currency = [
+        { value: 'USD', name: 'USD' },
+        { value: 'UAH', name: 'UAH' },
+    ];
+

```

Comment:
We have API for currency, you can check an example in dashboard page

Reply:
In this task, I only did UI. For the connection with the backend, I created another task and assigned it to myself. I will already connect the API there.